packages/web-components/test/agoric-wallet-connection.test.js

```diff
@@ -146,45 +162,40 @@ describe('AgoricWalletConnection', () => {
     it('goes to admin state once connected', async () => {
       iframeOnMessage('http://localhost:8000');
 
-      // Connecting happens instantly with the mock socket,
-      // just need to let the event loop run once.
-      await new Promise(resolve => setTimeout(resolve, 10));
+      await createStatePromise('bridged');
+      await nextFrame();
+      await nextFrame();
```

Comment:
These `await nextFrame()`s are still needed unfortunately. If I only have 1, it's flaky, but using 2 passed 50 times in a row :/